docs/userguide/dockerimages.md

```diff
@@ -268,25 +251,22 @@ example now for building our own Sinatra image for our development team.
     RUN apt-get update && apt-get install -y ruby ruby-dev
     RUN gem install sinatra
 
-Let's look at what our `Dockerfile` does. Each instruction prefixes a statement and is capitalized.
+Examine what your `Dockerfile` does. Each instruction prefixes a statement and
+is capitalized.
```

Comment:
strictly, it's not a requirement to capitalize (but we recommend doing for consistency)